src/RobotInterface/include/WalkingControllers/RobotInterface/Helper.h

```diff
@@ -62,7 +64,7 @@ namespace WalkingControllers
         iDynTree::VectorDynSize m_jointVelocitiesBounds; /**< Joint Velocity bounds [rad/s]. */
         iDynTree::VectorDynSize m_jointPositionsUpperBounds; /**< Joint Position upper bound [rad]. */
         iDynTree::VectorDynSize m_jointPositionsLowerBounds; /**< Joint Position lower bound [rad]. */
-
+         std::vector <bool> m_isJointModeStiffVector;
```

Comment:
Can you add the comment here? Using `/**< */`
Furthermore, you can save `std::vector<yarp::dev::InteractionModeEnum>` instead of  `std::vector <bool>`